examples/impute/plot_missing_values.py

```diff
@@ -75,11 +77,14 @@ def get_results(dataset):
 
     # Estimate the score after iterative imputation of the missing values
     estimator = make_pipeline(
-        make_union(IterativeImputer(missing_values=0, random_state=0),
+        make_union(IterativeImputer(missing_values=0,
+                                    random_state=0,
+                                    n_nearest_features=5),
                    MissingIndicator(missing_values=0)),
         RandomForestRegressor(random_state=0, n_estimators=100))
     iterative_impute_scores = cross_val_score(estimator, X_missing, y_missing,
```

Comment:
can you use a function to encapsulate creating the pipeline with a specified imputer and getting the scores? That way we might avoid similar errors here.

Reply:
Sure, can do.